widget/Toaster.js

```diff
@@ -76,10 +78,11 @@ define([
 			this.hide();
 
 			// place node as a child of body for positioning
-			this.ownerDocument.body.appendChild(this.domNode);
+			this.domNode.ownerDocument.body.appendChild(this.domNode);
```

Comment:
More evidence that something is wrong with the inheritance chain here, as this.ownerDocument is not defined, which is why I made the switch to at least get it working again...